src/server/api/interaction-step.js

```diff
@@ -9,6 +9,7 @@ export const resolvers = {
         "script",
         "answerOption",
         "answerActions",
+        "answerActionsData",
```

Comment:
I addressed this in a comment on the earlier, closed PR.  To address this concern this field will only be a name.   When processAction sees the name, it can dereference it.  External-system specific information will never leave the server.

Reply:
SHA 2839022be5b411465e83590d19ba9fd7babf8c0b